canonical-websites/www.ubuntu.com
# Update download page as per copy doc

## Done

Remove sections as specified in the [copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)

Variable changes are addressed in [this PR](https://github.com/canonical-websites/www.ubuntu.com/pull/2911)


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/desktop>
- Make sure content matches copy doc (note: versions numbers will be different as variable changes are in another PR)


## Issue / Card

Fixes #2867
